src/Utils/arm/request.ts

```diff
@@ -43,7 +44,8 @@ interface Options {
 // TODO: This is very similar to what is happening in ResourceProviderClient.ts. Should probably merge them.
 export async function armRequest<T>({ host, path, apiVersion, method, body: requestBody }: Options): Promise<T> {
   const url = new URL(path, host);
-  url.searchParams.append("api-version", apiVersion);
+  console.log(configContext);
```

Comment:
Not sure whether this is for dev purposes or cx-facing. If the latter, consider tracing somewhere (like notification console or something).